src/theory/arith/nl/transcendental/sine_solver.cpp

```diff
@@ -134,28 +134,6 @@ void SineSolver::checkInitialRefine()
           d_data->d_im.addPendingLemma(
               lem, InferenceId::ARITH_NL_T_INIT_REFINE, proof);
         }
-        {
-          // sine symmetry: sin(t) - sin(-t) = 0
-          Node lem = nm->mkNode(Kind::ADD, t, symn).eqNode(d_data->d_zero);
-          CDProof* proof = nullptr;
-          if (d_data->isProofEnabled())
-          {
-            proof = d_data->getProof();
-            Node tmplem =
-                nm->mkNode(Kind::ADD,
-                           t,
-                           nm->mkNode(
-                               Kind::SINE,
-                               nm->mkNode(Kind::MULT, d_data->d_neg_one, t[0])))
-                    .eqNode(d_data->d_zero);
-            proof->addStep(
-                tmplem, PfRule::ARITH_TRANS_SINE_SYMMETRY, {}, {t[0]});
```

Comment:
This is probably the only usage of this proof rule. Should we remove it?

Reply:
Let's wait.  Once the refactoring of this issue is done, I will revisit.